src/routes/events.js

```diff
@@ -0,0 +1,120 @@
+const express = require('express');
+const auth = require('../utils/auth');
+const sanitize = require('mongo-sanitize');
+const router = express.Router();
+const Events = require('../models/event');
+const User  =   require('../models/user');
+
+// @route    GET api/events
+// @desc     fetch all events
+// @access   Public
+router.get('/', async(req, res)=> {
+    try{
+        const events = await Events.find({ });
+        res.json(events);
+    }catch(err){
+        console.log(err.message);
+        res.status(500).send('Server Error');
+    }
+});
+
+
+// @route    GET api/events/:eventId
+// @desc     fetch single event details
+// @access   Public
+router.get('/:eventId', async(req, res)=> {
+    try{
+        const event = await Events.findOne({eid: req.params.eventId});
+        if(!event) {
+            res.status(404).send("Resource Not Found");
+        }
+        else {
+            return res.json(event);
+        }
+    }catch(err){
+        console.log(err.message);
+        res.status(500).send('Server Error');
+    }
+});
+
+// @route    GET api/events/:eventId/participants
+// @desc     fetch all participants of an event
+// @access   Public
+router.get('/:evendId/participants', async(req, res)=> {
+    try{
+        const eventParticipants = await Events.findOne({uid: req.params.eventId});
```

Comment:
findOne on event should be on eid i guess

Reply:
oh! my bad! I've changed that!